src/LinAlg/matrix.h

```diff
@@ -1392,9 +1400,11 @@ namespace utl //! General utility classes and functions.
                             unsigned int ofsx, int stridex,
                             unsigned int ofsy, int stridey)
   {
+    size_t size = std::min(this->size() / stridey, X.size() / stridex);
+    if (size == 0)
+      return *this;
```

Comment:
Is this early return necessary here? Since no blas-call involved or `ptr()` which I though was the problem (`data()` should be allowed also for empty vectors, al long you don't try to dereference the result.

Reply:
used data()